src/global/networkConstants.ts

```diff
@@ -182,6 +182,11 @@ export const tokenSymbol = {
 	CGT: 'CGT'
 };
 
+export const AssetsNetwork = {
```

Comment:
please add new scalable format as discussed

Reply:
`supportedAssets: [
			{
				name: 'dot-is-ded',
				tokenDecimal: 10,
				assetId: 12345
			}
		]`